src/components/ADempiere/Form/VPOS2/Options/Shipments/index.vue

```diff
@@ -155,16 +185,57 @@ export default defineComponent({
       })
     }
 
+    function selectLine(row, column, event) {
+      const { property } = column
+      if (property === 'quantity') {
+        row.isEditQty = true
+      }
+    }
+
+    function exitLine(row, column, event) {
+      const { property } = column
+      if (property === 'quantity') {
+        row.isEditQty = false
+      }
+    }
+
+    function updateQuantity(quantity) {
+      console.log(quantity, line.value.product.name)
```

Comment:
please remove this console.log

Reply:
https://github.com/solop-develop/frontend-core/pull/1568/commits/5b455b0b724f07ad2115182419ba4a3fa28e91ba